README.md

```diff
@@ -434,7 +434,7 @@ Then run `cargo build` or `cargo update && cargo build` for your project.
 
 * **derive**: Enables the custom derive (i.e. `#[derive(Clap)]`). Without this you must use one of the other methods of creating a `clap` CLI listed above
 * **suggestions**: Turns on the `Did you mean '--myoption'?` feature for when users make typos. (builds dependency `strsim`)
-* **color**: Turns on colored error messages. (builds dependency `termcolor`)
+* **color**: Turns on colored error messages. It does not turns on colored help, use `AppSettings::ColoredHelp` to have it. (builds dependency `termcolor`)
```

Comment:
> It does not turns on colored help, use `AppSettings::ColoredHelp` to have it.

You still have to enable it by setting `AppSettings::ColoredHelp`.

Reply:
> You still have to turn on colored help by setting AppSettings::ColoredHelp.

I don't think that is a better text since it does not imply that `ColoredHelp` is off by default.

Reply:
> still have to turn on

What part of that doesn't imply that it is off?

Reply:
```suggestion
* **color**: Turns on colored error messages. You still have to turn on colored help by setting `AppSettings::ColoredHelp`. (builds dependency `termcolor`)
```

Reply:
> What part of that doesn't imply that it is off?

The full sentence not so, that part yes. Either way, I just apply the change you suggested.